languages.toml

```diff
@@ -446,7 +446,7 @@ source = { git = "https://github.com/cstrahan/tree-sitter-nix", rev = "6b71a810c
 name = "ruby"
 scope = "source.ruby"
 injection-regex = "ruby"
-file-types = ["rb", "rake", "rakefile", "irb", "gemfile", "gemspec"]
+file-types = ["rb", "rake", "Rakefile", "irb", "Gemfile", "gemspec"]
```

Comment:
I've seen some `foo.rakefile` files in the wild. I'm not sure it's technically valid of if it's just allowed by case-insensitive file systems. Let's have both `rakefile` and `Rakefile` for anyone using the lowercase version (might as well do the same for the gemfile too)

Reply:
@the-mikedavis 
Thank you for your review.
I changed to both file type is available.

https://github.com/helix-editor/helix/commit/ea7a1604bb305499df66078112ba1ea805d6804f